package.json

```diff
@@ -44,16 +44,16 @@
     "package": "gulp package",
     "prepare": "gulp prepare",
     "prepareDemos": "gulp prepareDemos",
-    "publish": "gulp publish",
-    "publish:beta": "gulp publishBeta",
+    "publish": "npm ci && gulp publish",
+    "publish:beta": "npm ci && gulp publishBeta",
     "recompile": "gulp recompile",
     "release": "gulp gitCreateRC",
```

Comment:
What does this do? Does this also need an `npm ci`?

Reply:
No; this only does some `git` commands.